### ISSUE TITLE
Module specifier duplication not applying resolveModuleSource in System output

### ISSUE THREAD

**Issue by guybedford** [babel/babel#1342]
The module source looks like:

```javascript
import parseIRI from './iri';
import typeProcessors from './typeProcessors';
import * as qs from './qs';
import {
          clone,
          defaults,
          parseResponseHeaders,
          createResult,
          startsWith,
          callHandlers
        }
        from './helpers';
export default Pajax;
export * from './iri';
export * from './qs';
```

I'm setting a `resolveModuleSource` function which is providing a normalized value for all of these dependencies.

The resulting output looks like:

```javascript
System.register('github:n-fuse/pajax@0.1.7/pajax', ['npm:babel-runtime@5.1.11/helpers/create-class', 'npm:babel-runtime@5.1.11/helpers/class-call-check', 'npm:babel-runtime@5.1.11/helpers/define-property', 'npm:babel-runtime@5.1.11/core-js/promise', 'npm:babel-runtime@5.1.11/core-js/object/keys', 'github:n-fuse/pajax@0.1.7/iri', 'github:n-fuse/pajax@0.1.7/typeProcessors', 'github:n-fuse/pajax@0.1.7/qs', 'github:n-fuse/pajax@0.1.7/helpers', './iri', './qs'], function (_export) {
  // ...
});
```

That is, the duplicated names aren't being normalized properly.

Note it should also be fine to dedupe dependencies at this level, as long as the first use order is maintained.

**Comment by sebmck** [comment]
Actually, looks like this was because `resolveModuleSource` wasn't being called on `ExportAllDeclaration`s.

**Comment by guybedford** [comment]
:+1:

**Comment by sebmck** [comment]
Fixed as of 5.1.12, thanks!

**Comment by OrKoN** [comment]
Thanks @sebmck and @guybedford It works now
